lib/corpus.py

```diff
@@ -42,13 +42,24 @@ class CorpusParser(sax.handler.ContentHandler):
     is currently beeing read.
     """
 
-    def __init__(self, corpus: Corpus, path: str):
+    def __init__(self, corpus: Corpus, path: str, pretty_format_orth: bool = True):
```

Comment:
I would call it strip_orth to be more descriptive.. (the prettiness is arguable).

I request changes as to the default should preserve old behavior and therefore be false.

Reply:
I agree on using a better name.

But the default would be True? The current code does this stripping.

Reply:
> But the default would be True? The current code does this stripping.

Sorry, right.. I missed that and only saw the insertions.

Reply:
My proposal for the name of the flag would be `reformat_orth`. Functionality itself is OK I guess.